.github/workflows/release-to-staging.yml

```diff
@@ -0,0 +1,87 @@
+name: Release to Staging
+on:
+  pull_request:
+    branches:
+      - master
+      - hotfix-*
```

Comment:
+1 should we keep the convention of `hotfix/*` here? also, now that I wrote this I'm not sure we have a convention but that's something I'm used to ;)

Reply:
Good eye! With this change, I'm proposing that we change the hotfix flow to ALWAYS go to staging. If the PR is time critical, you can still merge to prod without waiting for staging to complete. For everything else, it's an extra chance to check your migration or code fix without delay!